Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -206,16 +204,6 @@ def sha256(arg = nil)
       @sha256 ||= arg
     end
 
-    def license(arg = nil)
```

Comment:
We don't want to remove the method entirely, as that would break Casks that still have a `license` stanza. Just replace the method body with `odeprecated "Hbc::DSL#license"`